File: SDKs/Aspose.Words-Cloud-SDK-for-Java/src/main/java/com/aspose/words/model/SplitDocumentResponse.java
package com.aspose.words.model;

import com.aspose.words.model.SplitDocumentResult;
public class SplitDocumentResponse {
  private SplitDocumentResult SplitResult = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getSplitResult
	 * Gets SplitDocumentResult
	 * @return SplitResult
	 */
  public SplitDocumentResult getSplitResult() {
    return SplitResult;
  }

	/**
	 * setSplitResult
	 * Sets SplitDocumentResult
	 * @param SplitResult SplitDocumentResult
	 */
  public void setSplitResult(SplitDocumentResult SplitResult) {
    this.SplitResult = SplitResult;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class SplitDocumentResponse {\n");
    sb.append("  SplitResult: ").append(SplitResult).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Words-Cloud-SDK-for-.NET/Aspose.Words.Cloud.Sdk/Model/SplitDocumentResponse.cs
namespace Aspose.Words.Cloud.Sdk.Model 
{
  using System;
  using System.Text;
  using System.Collections;
  using System.Collections.Generic;
  using System.Runtime.Serialization;
  using Newtonsoft.Json;
  using Newtonsoft.Json.Converters;

  /// <summary>
  /// 
  /// </summary>  
  public class SplitDocumentResponse : SaaSposeResponse 
  {                       
        /// <summary>
        /// Gets or sets SplitResult
        /// </summary>  
        public SplitDocumentResult SplitResult { get; set; }

        /// <summary>
        /// Get the string presentation of the object
        /// </summary>
        /// <returns>String presentation of the object</returns>
        public override string ToString()  
        {
          var sb = new StringBuilder();
          sb.Append("class SplitDocumentResponse {\n");
          sb.Append("  SplitResult: ").Append(this.SplitResult).Append("\n");
          sb.Append("}\n");
          return sb.ToString();
        }
    }
}
